KarhooUISDK/Routing/BookingFlowCoordinator.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  BookingFlowCoordinator.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 04.04.2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+final class BookingFlowCoordinator: KarhooUISDKSceneCoordinator {
```

Comment:
I am thinking if we should not consider increasing area of responsibility of this coordinator to manage not only the booking process, but the whole user flow in general. What are your thoughts on that? 
Alternatively, we could create an additional Coordinator to handle this flow, so manage BookingFlow, Rides, Allocation, Tracking etc.

Reply:
I've renamed the BookingFlowCoordintator to MainFlowCoordinator. It will handle all user flows including booking, rides list, track driver and any other flow that may appear in the future.